app/javascript/App/bundles/CoinShow/Fundamentals.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react'
+import Icon from '~/bundles/common/components/Icon'
+import { Grid, withStyles, createStyles } from '@material-ui/core'
+import {
+  formatValueWithCurrency,
+  formatValue,
+} from '~/bundles/common/utils/numberFormatters'
+
+interface Props {
+  coinObj: any
+  currency: string
+  classes: any
+}
+
+const styles = (theme) =>
+  createStyles({
+    title: {
+      color: 'rgba(0,0,0,0.38)',
+      marginBottom: '4px',
+      fontSize: '14px',
+      lineHeight: '22px',
+    },
+    value: {
+      fontWeight: 'bold',
+      color: '#555',
+      marginBottom: '4px',
+      fontSize: '14px',
+      lineHeight: '22px',
+    },
+  })
+
+class Fundamentals extends React.Component<Props, {}> {
```

Comment:
It may be worth being a bit more descriptive with these component names so it's easier to identify what it does. E.g. `FundamentalsTable`